app/controllers/application_controller.rb

```diff
@@ -16,6 +16,9 @@ class ApplicationController < ActionController::Base
 
   protected
 
+  def track_multivariate_conversion(impression_id)
```

Comment:
Is this intentionally empty? May be add a comment if so?

Reply:
woops... its an artifact. will remove.